files/image_config/sysctl/sysctl-net.conf

```diff
@@ -17,12 +17,12 @@ net.ipv4.conf.all.arp_notify=1
 net.ipv4.conf.all.arp_ignore=2
 net.ipv4.neigh.default.base_reachable_time_ms=1800000
 net.ipv6.neigh.default.base_reachable_time_ms=1800000
-net.ipv4.neigh.default.gc_thresh1=1024
-net.ipv6.neigh.default.gc_thresh1=1024
-net.ipv4.neigh.default.gc_thresh2=2048
-net.ipv6.neigh.default.gc_thresh2=2048
-net.ipv4.neigh.default.gc_thresh3=4096
-net.ipv6.neigh.default.gc_thresh3=4096
+net.ipv4.neigh.default.gc_thresh1=32000
```

Comment:
This is a pretty big value change from existing value. If we keep these many entries without cleanup, it can cause some TABLE full issues.

Reply:
The garbage collector will not run if there are fewer than this number of entries in the cache - which was 1024 before this change . So it means if we have more than 1024 entries the garbage collector will start purging the entries in arp cache. Because of  purging of arp entries from kernel we will not be reaching higher scale.  In order to overcome it, we will need to set it to a much higher value i.e. which would be near to ASIC capability . That is why ipv4 thresh1 is set to 32k .